src/app/settings/payment.component.ts

```diff
@@ -61,8 +49,11 @@ export class PaymentComponent implements OnInit {
     private stripeCardNumberElement: any = null;
     private stripeCardExpiryElement: any = null;
     private stripeCardCvcElement: any = null;
+    private theme: any = null;
+    private StripeElementStyle: any;
+    private StripeElementClasses: any;
 
-    constructor(private platformUtilsService: PlatformUtilsService, private apiService: ApiService) {
+    constructor(private platformUtilsService: PlatformUtilsService, private apiService: ApiService, private storageService: StorageService) {
```

Comment:
Probably need to bump this down to the next line. See the rule below:

![Screen Shot 2021-08-05 at 3 44 20 pm](https://user-images.githubusercontent.com/31796059/128297544-35480a9c-8b16-449f-9754-9b229ba499a1.png)